C_CPP/Data Structures/Standard Template Library/MAPS.md

```diff
@@ -0,0 +1,3 @@
+## video for maps in c++ STL
+
+[maps in c++ stl by @Op-panda of batch-6 ] --> (https://drive.google.com/file/d/18Pe8z0WXx6TqCF6lNzoaymJ9AQLGbln0/view?usp=sharing)
```

Comment:
Make relative link with embedded picture, for eg:
[![Maps in C++ STL](_link to picture/screenshot of video_)](_link to video_)